README.md

```diff
@@ -1,239 +1,158 @@
 # Auth0 Java MVC Commons
 
+Easily add authentication with Auth0 to your Java Servlet applications.
```

Comment:
Let's find an alternative for "easily", as mentioned in Slack.

Reply:
The one-liner we have in the doc is:

> Auth0 SDK to add authentication to your Java Servlet applications

Reply:
lol I updated that one-liner in the doc, forgot to push the change here. Pushed a new commit to fix.